docs/README.md

```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
 https://www.cyberbotics.com/doc/guide/index?version=remidhum:reference_proto
 ```
 
-## Run the doc locally
+## So as to Run the doc locally,follow these steps:
 
-Set the terminal to the 'docs' directory.
+1.Set the terminal to the 'docs' directory.
```

Comment:
A space should be added after the dot.

Reply:
Still missing this one.

Reply:
I think I've added a space after the dot but I'll check it again.
Thanks for the review .

Reply:
I've added the spaces after the dot, thanks for identifying this .